locale/pl/LC_MESSAGES/client.po

```diff
@@ -412,6 +412,8 @@ msgid ""
 "<a href=\"https://support.mozilla.org/kb/im-having-problems-with-my-firefox-"
 "account\">Help</a>"
 msgstr ""
+"<a href=\"https://support.mozilla.org/pl/kb/rozwiazywanie-problemow-z-kontem-"
```

Comment:
I wasn't sure what to do here. The redirect does work, so I can revert to the original URL if you want.